src/Framework/Framework/ViewModel/Validation/ValidationErrorFactory.cs

```diff
@@ -20,22 +21,25 @@ namespace DotVVM.Framework.ViewModel.Validation
 {
     public static class ValidationErrorFactory
     {
-        public static ViewModelValidationError AddModelError<T, TProp>(this T vm, Expression<Func<T, TProp>> expr, string message)
-            where T : class, IDotvvmViewModel =>
-            AddModelError(vm, expr, message, vm.Context);
+        public static ViewModelValidationError AddModelError<T>(this T vm, string message)
+            where T : class, IDotvvmViewModel
+            => vm.Context.AddModelError(message);
```

Comment:
In this case, the error should be bound to the passed object
```suggestion
            => vm.Context.AddModelError(vm, a => a, message);
```